common/utils/get-focusable-elements.ts

```diff
@@ -1,12 +1,12 @@
 // https://gomakethings.com/how-to-get-the-first-and-last-focusable-elements-in-the-dom/
 function getFocusableElements<E extends Element = Element>(
-  el: HTMLElement
+  el: HTMLElement,
+  keepNegativeTabIndex?: boolean
```

Comment:
i think this is from [another PR](https://github.com/wellcomecollection/wellcomecollection.org/pull/10297)?